lldb/test/API/lang/swift/macro/TestSwiftMacro.py

```diff
@@ -42,8 +42,8 @@ def testDebugging(self):
         thread.StepInto()
         # This is the expanded macro source, we should be able to step into it.
         self.expect('reg read pc', substrs=[
-            '[inlined] freestanding macro expansion #1 of stringify',
-            '13testStringify'
+            '[inlined] freestanding macro expansion #1 of stringify in module a file main.swift line 5 column 11',
+            '__swiftmacro_1a0015mainswift_tzEGbfMX4_10_9stringifyfMf_'
```

Comment:
```suggestion
            'stringify'
```
This should suffice for what we're interested in here, and should be more future proof.

Reply:
@adrian-prantl did you also mean removing the `in module a file...`?

Reply:
no, just the details of the mangled name that are more likely to change